setup.py

```diff
@@ -180,6 +180,7 @@ def package_files(prefix: str, directory: str, extension: str):
 ]
 
 extra_deps['streaming'] = [
+    'mosaicml-streaming==0.0.1a0',
```

Comment:
```suggestion
    'mosaicml-streaming',
```

I don't suggest we pin this just yet since we are still prepping the `mosaicml/streaming` repo.  We should pin it or at least restrict it a bit once we finish prepping the streaming repo in a subsequent PR.

Reply:
Done